sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaId.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Data.SchemaRegistry.Models
+{
+    /// <summary>
+    /// JSON Object received from the registry containing schema identifiers.
```

Comment:
The description was copy-paste from the generated swagger type (thus, the description for this type in the swagger). However, this type is not public, so users would never see it. Since the description is duplicated, I've removed it.